src/test/java/com/ays/admin_user/controller/AdminUserRegisterApplicationControllerTest.java

```diff
@@ -281,4 +283,66 @@ void givenValidAdminUserRegisterApplicationId_whenAdminUserApplicationFound_then
                         .isNotEmpty());
     }
 
+    @Test
+    void givenValidAdminUserRegisterApplicationRejectRequest_whenRejectingAdminUserRegisterApplication_thenReturnSuccess() throws Exception {
+
+        // Given
+        String mockId = AysRandomUtil.generateUUID();
+        AdminUserRegisterApplicationRejectRequest mockRequest = new AdminUserRegisterApplicationRejectRequestBuilder()
+                .withValidFields()
+                .build();
+
+        // When
+        Mockito.doNothing().when(adminUserRegisterApplicationService).rejectRegistrationApplication(mockId, mockRequest);
+
+        // Then
+        String endpoint = BASE_PATH.concat("/registration-application/").concat(mockId).concat("/reject");
+        AysResponse<Void> mockAysResponse = AysResponse.SUCCESS;
+
+        mockMvc.perform(AysMockMvcRequestBuilders
+                        .post(endpoint, mockSuperAdminToken.getAccessToken(), mockRequest))
+                .andDo(MockMvcResultHandlers.print())
+                .andExpect(AysMockResultMatchersBuilders.status().isOk())
+                .andExpect(AysMockResultMatchersBuilders.time()
+                        .isNotEmpty())
+                .andExpect(AysMockResultMatchersBuilders.httpStatus()
+                        .value(mockAysResponse.getHttpStatus().getReasonPhrase()))
+                .andExpect(AysMockResultMatchersBuilders.isSuccess()
+                        .value(mockAysResponse.getIsSuccess()))
+                .andExpect(AysMockResultMatchersBuilders.response()
+                        .doesNotExist());
+
+        Mockito.verify(adminUserRegisterApplicationService, Mockito.times(1))
```

Comment:
Aşağıdaki gibi suggestion kısmını da ekleyebilir miyiz?
```suggestion

        // Verify
        Mockito.verify(adminUserRegisterApplicationService, Mockito.times(1))
```

Reply:
Düzeltildi